components/Layout/Navigation/Sidebar.tsx

```diff
@@ -15,7 +19,7 @@ export default function Sidebar({ isOpen, toggle }) {
   )
 }
 
-const SidebarContainer = styled.aside`
+const SidebarContainer = styled.aside<SideBarProps>`
   position: fixed;
   z-index: 999;
```

Comment:
Why do you need such a high `z-index` value?
Try using numbers between 1-10 to keep it in hand. Otherwise you can add unlimited 9's to have the higher value, but how do you then know which ones are the highest.